internal/pkg/action/operator_uninstall.go

```diff
@@ -140,15 +137,23 @@ func (u *OperatorUninstall) deleteObjects(ctx context.Context, objs ...controlle
 	return waitForDeletion(ctx, u.config.Client, objs...)
 }
 
-func (u *OperatorUninstall) getInstallPlanResources(ctx context.Context, installPlanKey types.NamespacedName) (crds, csvs, others []controllerutil.Object, err error) {
-	installPlan := &v1alpha1.InstallPlan{}
-	if err := u.config.Client.Get(ctx, installPlanKey, installPlan); err != nil {
-		return nil, nil, nil, fmt.Errorf("get install plan: %v", err)
+func (u *OperatorUninstall) getInstallPlanForCSV(ctx context.Context, installedCSV, ipNamespace string) (*v1alpha1.InstallPlan, error) {
+	installPlanList := &v1alpha1.InstallPlanList{}
+	if err := u.config.Client.List(ctx, installPlanList, client.InNamespace(ipNamespace)); err != nil {
+		return nil, fmt.Errorf("error getting install plan resources from namespace %s", ipNamespace)
```

Comment:
Nit just to match existing error formatting:
```suggestion
		return nil, fmt.Errorf("list install plans in namespace %q: %v", ipNamespace, err)
```